app/assets/javascripts/sitewide/file_preview.js

```diff
@@ -32,17 +32,25 @@
         link.attr('data-no-turbolink', true);
         link.attr('data-status', 'asset-present');
         if (data['type'] === 'image') {
-          modal.find('.file-preview-container')
-               .append($('<img>')
-                 .attr('src', data['large-preview-url'])
-                 .attr('alt', name)
-                 .click(function(ev) {
-                   ev.stopPropagation();
-                 })
-               );
+          if(data['processing']) {
+            $(data['processing-url'])
```

Comment:
I'm just thinking loud, we have 2 ways of displaying "progress" in the application: by displaying this `medium/processing.gif` image, and by using `spinnerjs` library (`.spin()`). 

I think, visually, the spinner library's spinner looks nicer. Is it possible to be implemented here as well, on the image preview?